shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -466,9 +466,22 @@
   );
 }
 
-bool FlutterPlatformViewsController::SubmitFrame(GrDirectContext* gr_context,
-                                                 std::shared_ptr<IOSContext> ios_context,
-                                                 std::unique_ptr<SurfaceFrame> frame) {
+bool FlutterPlatformViewsController::SubmitFrame(
+    GrDirectContext* gr_context,
+    std::shared_ptr<IOSContext> ios_context,
+    std::unique_ptr<SurfaceFrame> frame,
+    const std::shared_ptr<fml::SyncSwitch>& gpu_disable_sync_switch) {
+  bool result = false;
+  gpu_disable_sync_switch->Execute(
+      fml::SyncSwitch::Handlers().SetIfTrue([&] { result = false; }).SetIfFalse([&] {
```

Comment:
Is the `SetIfTrue` handler necessary?  `result` is already set to `false` by default

Reply:
Nope, it shouldn't be necessary.  Once I get things sorted out I'll try removing it.

Reply:
Oh, shoot I forgot to do this before it landed.  I was in a rush.  It's probably not a big deal, if you want it gone I can do it in a separate PR.  Sorry.